File: C/con7.c
#include <stdio.h>
int main(int argc, char *argv[]) {
    int l,b;
    printf("enter l and b\n");
    scanf("%d %d",&l,&b);
    printf("area of rectangle is=%d\n",l*b);
    if(l==b){
        printf("This is square");
    }else {
        printf("This is not square");
    }
}

File: PYTHON/conditionProblems/con7.py
"""
  simple Hello world program
  By Itfyme.com
"""

def operation():
    print("enter l and b")
    l=int(input())
    b=int(input())
    print("area of rectangle",l*b)
    if l==b:
        print("This is square")
    else:
        print("This is not square")
 
operation()
